components/dialog/overview.md

```diff
@@ -53,6 +53,12 @@ The Dialog component and its [predefined options]({%slug dialog-predefined%}) ai
 
 Predefined Dialogs are styled substitutes to the standard browser dialogs - confirm, alert and prompt. [Read more about the Blazor Predefined Dialogs]({%slug dialog-predefined%}).
 
+## Integration with other components
```

Comment:
This section is important, but not more than the built-in Dialog features. Consider moving it before or after Events.